public/app/plugins/panel/geomap/layers/data/StyleEditor.tsx

```diff
@@ -178,6 +187,24 @@ export const StyleEditor: FC<StandardEditorProps<StyleConfig, StyleEditorOptions
           }
         />
       </Field>
+      <Field label={'Angle in Degrees'}>
```

Comment:
Do we also want to include a version of this new UI for geojson layer (when data has points?) If that is the case it probably would make sense to rebase this on latest master and include these changes as part of the `featuresHavePoints` conditional rendering block